app/models/concerns/twitter_user/builder.rb

```diff
@@ -0,0 +1,114 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Builder
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def build_by_user(user, attrs = {})
+      build_relation = attrs.has_key?(:build_relation) ? attrs.delete(:build_relation) : false
+      tu = new(attrs) do |tu|
+        tu.uid = user.id
+        tu.screen_name = user.screen_name
+        tu.user_info = user.slice(*TwitterUser::PROFILE_SAVE_KEYS).to_json # TODO check the type of keys and values
```

Comment:
__[rubocop]__
* Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem. [:ref](https://github.com/bbatsov/ruby-style-guide#annotate-keywords)
* Line is too long. [115/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)